src/dragonball/src/dbs_virtio_devices/src/vsock/device.rs

```diff
@@ -32,6 +32,57 @@ const VSOCK_CONFIG_SPACE_SIZE: usize = 8;
 const VSOCK_AVAIL_FEATURES: u64 =
     1u64 << uapi::VIRTIO_F_VERSION_1 | 1u64 << uapi::VIRTIO_F_IN_ORDER;
 
+/// Vsock-related metrics.
+#[derive(Default, Serialize)]
+pub struct VsockDeviceMetrics {
+    /// Number of times when handling events on a vsock device.
```

Comment:
Can u add metrics to trace local init connection or peer init connection?

Reply:
Of course, I've added them.